app/services/github_service.rb

```diff
@@ -0,0 +1,42 @@
+class GithubService
+  def initialize(token)
+    @token = token
+    @_connection = Faraday.new("https://api.github.com/")
+  end
+
+  def starred
+    parser(connection.get "/user/starred?access_token=#{@token}")
```

Comment:
@caward12 Maybe try to move toward using Faraday's ability to pass params as a hash instead of using string interpolation.